src/python/turicreate/data_structures/sframe.py

```diff
@@ -1603,13 +1603,18 @@ def read_json(cls,
         [3 rows x 1 columns]
         """
```

Comment:
I don't think `url` is the right name for the first parameter any more. Maybe we should call it `url_or_string`.